src/background/grant/createQuote.ts

```diff
@@ -0,0 +1,44 @@
+import { AxiosInstance } from 'axios'
+
+import { getHeaders } from './getHeaders'
+
+type TCreateQuote = (_params: {
+  receiver: string
+  walletAddress: any
```

Comment:
What actually gets passed in here? Let's avoid using `any` and create some small types until we decide how we should use the OpenAPI types.